README.md

```diff
@@ -190,6 +194,28 @@ bundle exec rails destroy logidze:model Post
 
 **IMPORTANT**: If you use non-UTC time zone for Active Record (`config.active_record.default_timezone`), you MUST always infer log timestamps from a timestamp column (e.g., when back-filling data); otherwise, you may end up with inconsistent logs ([#199](https://github.com/palkan/logidze/issues/199)). In general, we recommend using UTC as the database time unless there is a very strong reason not to.
 
+### Sequel support
+
+All mentioned generators also optionally support Sequel.
```

Comment:
We should mention here that the current Sequel support is more like "Sequel within Rails" support, since we still re-use a lot of Rails features. Splitting the library to work with pure Sequel (and without any Rails stuff) will be a next step (some day in the future).